exploring-ship-logbooks/word_count.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import math
+
+def word_count(data, columns, key_words):
+    """
+    Generates a list containing the number of counts of each key word in
+    each logbook entry.
+
+    Inputs:
+        - data: raw logbook data
+        - columns: columns of data frame "data" to include in the search
+                  (typical value is ['CargoMemo', 'LifeOnBoardMemo', 'OtherRem'])
+        - key_words: words to search for.
+            Note: a word will be counted if it contains a key word.
+                  i.e. "captain" will be counted in a search for "cap"
+
+    Outputs:
+        - word_count_list: a list of dictionaries with the same length as data.
+                           each dictionary contains an entry for each key word
+                           with the number of times that word appeared in an
+                           entry.
+
+    """
+
+    # read each row in data
+    word_count_list = []
```

Comment:
Can you initialize a dictionary this way? I am just curious. Normally I have seen {} or dict().